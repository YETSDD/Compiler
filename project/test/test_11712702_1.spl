int test_11712702_1()
{
  int i = 0xffg, j = 1;
  return i

}