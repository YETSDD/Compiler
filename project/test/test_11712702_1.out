Error type A at Line 3 : Unknown lexeme 0xffg
Error type B at line 4 : Missing semicolon ';'